benchmarks/quac/f1.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import re
+import string
+import nltk
+
+from collections import Counter
+from statistics import mean
+from typing import List
+from nltk.corpus import stopwords
+
+from inspect_ai.scorer import scorer, mean, stderr, Score, Target, CORRECT, INCORRECT
+from inspect_ai.solver import (
+    TaskState,
+)
+
+logger = logging.getLogger(__name__)
+nltk.download("stopwords", quiet=True)
+
+STOP_WORDS = set(stopwords.words("english"))
+MIN_HUMAN_F1 = 0.4  # 0.4 was used in the paper
+
+
+@scorer(metrics=[mean(), stderr()])
+def f1_scorer():
+    async def score(state: TaskState, targets: Target) -> Score:
+        prediction = state.output.completion
+
+        if prediction.upper() == "CANNOTANSWER":
+            return Score(
+                value=CORRECT
+                if any(t.upper() == "CANNOTANSWER" for t in targets)
+                else INCORRECT,
+                answer="CANNOTANSWER",
+            )
```

Comment:
I'm currently explicitly checking for CANNOTANSWER. Maybe that can be removed / simplified though. Need to be tested